### PR TITLE
Add charter

### PR DESCRIPTION
**Pull request by cartersocha**
Create charter for security sig

### CODE REVIEW

File: sig-security-charter.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+## Scope
+
+SIG Security covers horizontal security initiatives for the OpenTelemetry (OTel)
+project and is focused on improving the security of the project across all
+components.
+
+This includes preparing the project for regular security audits, improving the
+vulnerability management process, producing cross-cutting security
+documentation, serving as a security community touch point, managing the
+non-emnargoed (public) vulnerability process, default repo configurations, and
```
Comment (jpkrohling) [inline sig-security-charter.md:12]:
```suggestion
non-embargoed (public) vulnerability process, default repo configurations, and
```

File: sig-security-charter.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+As a process-oriented SIG, SIG Security does not directly own OpenTelemetry component
+code.
+
+### In scope
+
+#### Vulnerability Management Process
+
+Work with the OpenTelemetry Technical Committee (TC) to define the processes for fixing and disclosing vulnerabilities. For example:
+
+- The default level for OTel repo Github security configuration and tooling
```
Comment (jpkrohling) [inline sig-security-charter.md:24]:
```suggestion
- The default level for OTel repo GitHub security configuration and tooling
```

File: sig-security-charter.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+- Post-announcement follow-ups, such as additional fixes, mitigations, preventions or documentation after a vulnerability is made public
+- Distributor announcement policies, such as timelines, criteria for joining the list, etc.
+- How, when and where vulnerabilities are announced
+
+#### Security Community Management and Outreach
+
+Provide an entry point to the OTel community for new security-minded contributors, as well as a meeting point to discuss security themes and issues within OTel, including:
+
+- Answer security questions from inexperienced users (that don't know what SIG to go to), and identify common questions or issues as areas for improvement.
+- Provide an "entry point" for new contributors interested in security. Route these new contributors to other SIGs when they have more specific goals (e.g. SIG Node for container isolation).
```
Comment (jpkrohling) [inline sig-security-charter.md:37]: nit: some lines are being wrapped, other are not. While I prefer to not wrap the lines automatically (I can have my code editor do that for me according to my taste), I don't mind forcing wraps, as long as it's consistent across the whole document :-)

File: sig-security-charter.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+Manage recurring CNCF security audits and follow up on issues. Coordinate vendors to perform the audit and publish the findings. Follow up on issues with the affected SIG and help coordinate resolution, which can include:
+
+- Helping to prioritize the fixes, possibly by recruiting from SIG Security (while acknowledging that the ultimate authority in deciding whether and how to fix an issue lies with the responsible SIG).
+- Documenting mitigations, workarounds, or caveats, especially when the responsible SIG decides not to fix a reported issue.
+
+### Out of scope
+
+SIG Security’s scope does not include:
+
+- Developing any new telemetry signals, SDKs, APIs etc.
```
Comment (jpkrohling) [inline sig-security-charter.md:56]: I would explicitly state that we are not in the business of defining semantic conventions for security domains.

File: sig-security-charter.md (hunk 3 above)
Comment (cartersocha) [reply sig-security-charter.md]: Habit from working with Reiley's team haha. Updated

**Review by codeboten — COMMENTED** [review]
Assuming the deletion was not intentional

Screenshot 2023-07-18 at 2 46 44 PM

File: sig-security-charter.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+SIG Security covers horizontal security initiatives for the OpenTelemetry (OTel)
+project and is focused on improving the security of the project across all
+components.
+
+This includes preparing the project for regular security audits, improving the
+vulnerability management process, producing cross-cutting security
+documentation, serving as a security community touch point, managing the
+non-embargoed (public) vulnerability process, default repo configurations, and
+defining the bug bounty.
```
Comment (codeboten) [inline sig-security-charter.md:13]:
```suggestion
This includes:
* preparing the project for regular security audits
* improving the vulnerability management process
* producing cross-cutting security documentation
* serving as a security community touch point
* managing the non-embargoed (public) vulnerability process
* default repo configurations
* defining the bug bounty
```

File: sig-security-charter.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+- Developing any new telemetry signals, SDKs, APIs etc.
+- Defining security semantic conventions
+- Security audit for other CNCF projects (e.g., etcd, CoreDNS, CRI-O,
+  containerd) (Belongs to the CNCF’s SIG Security.)
+- Any projects outside of the OpenTelemetry project and community repos
+- Vendor or distribution specific guidelines and vulnerability coverage
+- Recommendations or endorsements of specific commercial product vendors or
+  cloud providers
+- Private vulnerability response (belongs to the PSC), including:
```
Comment (codeboten) [inline sig-security-charter.md:79]: What's PSC?

File: sig-security-charter.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+  responsible SIG decides not to fix a reported issue.
+
+### Out of scope
+
+SIG Security’s scope does not include:
+
+- Developing any new telemetry signals, SDKs, APIs etc.
+- Defining security semantic conventions
+- Security audit for other CNCF projects (e.g., etcd, CoreDNS, CRI-O,
+  containerd) (Belongs to the CNCF’s SIG Security.)
```
Comment (codeboten) [inline sig-security-charter.md:74]: Is there a link to CNCF's SIG?

```suggestion
- Security audit for other CNCF projects (e.g., etcd, CoreDNS, CRI-O,
  containerd). These belong to the CNCF’s SIG Security.
```

Comment (cartersocha) [reply sig-security-charter.md]: https://github.com/cncf/tag-security

File: sig-security-charter.md (hunk 6 above)
Comment (cartersocha) [reply sig-security-charter.md:82]: In this case I think we would put repo owners here

**Comment by cartersocha** [comment]
Should be ready for final reviews @codeboten @jpkrohling

**Review by jpkrohling — APPROVED** [review]

File: README.md [hunk 8]
```diff
@@ -1,13 +1,19 @@
 # Security Special Interest Group
 
-Covers horizontal security initiatives for the OpenTelemetry project, including regular security audits, the vulnerability management process, cross-cutting security documentation, and security community management.
+Covers horizontal security initiatives for the OpenTelemetry project, including
+regular security audits, the vulnerability management process, cross-cutting
+security documentation, and security community management.
 
-The [charter](https://docs.google.com/document/d/1fHuqKLeWwKYf5KBCpySqFTwCKuD2i8kK2D2RF3sVh4g/edit) draft defines the scope and governance of the Security Special Interest Group.
+The
+[charter](https://docs.google.com/document/d/1fHuqKLeWwKYf5KBCpySqFTwCKuD2i8kK2D2RF3sVh4g/edit)
```
Comment (codeboten) [inline README.md:8]: should this point to the charter in the repo?

**Review by codeboten — APPROVED** [review]

File: README.md (hunk 8 above)
Comment (cartersocha) [reply README.md:8]: Yeah I’ll update it. Legacy from when I had a readme pr open. Was planning on just adding the charter subsequently

Comment (cartersocha) [reply README.md]: Updated

**Pull request merged**